vendor/github.com/go-openapi/swag/string_bytes.go

```diff
@@ -0,0 +1,8 @@
+package swag
+
+import "unsafe"
```

Comment:
do we know where and how this is used? just want to make sure nothing is sharing byte data in ways that break our use of shared openapi objects

Reply:
(sorry... wrong PR, I meant to ask this on https://github.com/kubernetes/kubernetes/pull/126787)